src/js/dialogs/confirmShowSolution.js

```diff
@@ -53,7 +53,7 @@ exports.dialog = {
     type: 'ModalAlert',
     options: {
       markdowns: [
-        '## ¿Estás seguro de que quiere ver la solución?',
+        '## ¿Estás seguro de que quieres ver la solución?',
```

Comment:
👍 I was going to suggest this change in #745 but I assumed it was a local variation. 😄